src/web/client/common/remoteSaveProvider.ts

```diff
@@ -32,32 +34,32 @@ export async function saveData(
         }
         requestBody = JSON.stringify(data);
     } else {
-        sendAPIFailureTelemetry(requestUrl, 0, BAD_REQUEST); // no API request is made in this case since we do not know in which column should we save the value
+        sendAPIFailureTelemetry(requestUrl, entityName, httpMethod.PATCH, 0, BAD_REQUEST); // no API request is made in this case since we do not know in which column should we save the value
```

Comment:
I would consider NOT using the sendAPIFailureTelemetry event for this kind of error since there's no actual API being called yet.
Record this simply as another error or perhaps throw so the caller knows this is an invalid input?
this seems more like a caller having invalid parameters vs an API issue.